agate/agate/queue_reading/ingestion_updater.py

```diff
@@ -0,0 +1,79 @@
+import logging
+from agate.models import IngestionAttempt
+from agate.forms import IngestionAttemptForm
+
+logger = logging.getLogger(__name__)
+
+
+class IngestionUpdater:
+    """
+    This class interprets a dictionary (as may come from a rabbitMQ message)
+    and creates or updates an IngestionAttempt appropriately.
+    """
+
+    @classmethod
+    def update(cls, data: dict, stage: str):
+        """
+        Use a dictionary to update items in the database
+        """
+        try:
+            uuid = data["uuid"]
+        except KeyError:
+            logger.fatal("no uuid")
+            return
```

Comment:
My instinct is that this try-except should be implemented in whatever calls `update`, rather than here. Is returning `None` really a useful behaviour here? (I can entirely believe that it is but I feel like this block pretends to handle and exception but doesn't really.)